numberFormat/intructions.html

```diff
@@ -0,0 +1,7 @@
+Format any integer provided into a string with "," (commas) in the correct places.
+
+Example:
+
+For n = 100000 the function should return '100,000';
```

Comment:
I would suggest to make the instructions more readable by renaming **n** into a meaningful name